xformers/components/attention/csrc/cuda/mem_eff_attention/kernel_backward.h

```diff
@@ -130,7 +178,7 @@ struct AttentionBackwardKernel {
     }
 
     __host__ dim3 getBlocksGrid() const {
-      return dim3(1, 1, num_batches);
+      return dim3(1, num_heads, num_batches);
```

Comment:
interesting choice for the shape of the grid. I don't think it might matter for speed, just curious